ed25519_go1_13.go

```diff
@@ -0,0 +1,86 @@
+// +build go1.13
+
+package jwt
+
+import (
+	"errors"
+
```

Comment:
Everything looks good to me, I shall merge this tomorrow, so thanks for your contribution!

Just a nitpick, could you remove this extra line? I like to have the standard library grouped in the imports. 👼